Mathlib/Algebra/Order/BigOperators/Group/List.lean

```diff
@@ -174,5 +174,17 @@ variable [CanonicallyOrderedCommMonoid M] {l : List M}
     exact le_self_mul
   · simp [take_of_length_le h, take_of_length_le (le_trans h (Nat.le_succ _))]
 
+@[to_additive] theorem le_prod_of_mem
```

Comment:
This looks like an alternate statement of `single_le_prod`, and as such I think they should both have docstrings referencing the other. It also looks like this name is inconsistent with that one, although I personally prefer this name

Reply:
They have quite different hypotheses. I'll add cross-references in doc-strings.